src/Models/RacingGame.js

```diff
@@ -0,0 +1,71 @@
+import { Car } from './';
+import { MESSAGE, RACING_GAME, UTIL } from '../constants';
+import {
+  getRacingResult,
+  getRandomIntInRange,
+  findMaxDistance,
+  findWinners,
+  parseGameResult,
+} from '../utils';
+
+export class RacingGame {
+  #cars;
+  #gameProgress;
+  #gameResult;
+
+  constructor(carNames, totalRounds) {
+    this.#cars = [];
+    this.#gameProgress = RACING_GAME.PROGRESS_TITLE;
+    this.#settingRacingGame(carNames, totalRounds);
+  }
+
+  #settingRacingGame(carNames, totalRounds) {
+    for (let carName of carNames) this.#cars.push(new Car(carName));
+    this.#raceWithTotalRounds(totalRounds);
+  }
+
+  #raceWithTotalRounds(totalRounds) {
+    for (let round = 0; round < totalRounds; round++) {
+      this.#cars.forEach((car) => this.#randomCarMovement(car));
+      this.#recordGameProgress();
+    }
+
+    this.#checkWinner();
+  }
+
+  #randomCarMovement(car) {
+    const randomInt = getRandomIntInRange(
+      UTIL.RANDOM_INT_MIN,
+      UTIL.RANDOM_INT_MAX
+    );
+
+    if (RACING_GAME.MOVEMENT_THRESHOLD <= randomInt) car.advance();
```

Comment:
긍정 구문을 조건문으로 분기시키는 것보다, `early return`을 통해 구문을 정리해보면 어떨까요?

```suggestion

    if (RACING_GAME.MOVEMENT_THRESHOLD > randomInt) return;
    car.advance();
```